libs/hwManager/manufacturers/index.js

```diff
@@ -1,7 +1,7 @@
-import ledger from './ledger';
-import trezor from './trezor';
+import Ledger from './ledger';
```

Comment:
You don't need this file. Whenever you need to import both files in another file just import the whole folder like this 
`import * as libs from 'libs/hwManager/manufacturers'`.

Reply:
base on the new structure we need it, so at least for now is better in the way it is, in case this will change we can use this solution, thx anyway